examples/js/exporters/GLTFExporter.js

```diff
@@ -1,6 +1,17 @@
 //------------------------------------------------------------------------------
 // Constants
 //------------------------------------------------------------------------------
+import {
+	RedFormat,
+	RedIntegerFormat,
+	RGBAFormat,
+	RGBAIntegerFormat,
+	RGBEFormat,
+	RGBFormat,
+	RGBIntegerFormat, RGFormat,
+	RGIntegerFormat
+} from "../../../build/three.module";
+
```

Comment:
In `examples/js` files, the usage of `three.module` is invalid. Since they are global scripts, you have to use the `THREE` namespace. Besides, the module file (`examples/jsm`) is generate via `node utils/modularize.js`.

Reply:
Woops, IDE auto-import shenanigans. Will fix that asap

Reply:
Fixed 9afb6ab